frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -50,11 +57,32 @@ export const TopologyDataController: React.FC<TopologyDataControllerProps> = ({
   application,
   cheURL,
   resourceList,
+  serviceBinding,
 }) => {
   const { resources, utils } = getResourceList(namespace, resourceList);
+  resources.push(
+    {
+      isList: true,
+      kind: referenceForModel(ClusterServiceVersionModel),
+      namespace,
+      prop: 'clusterServiceVersion',
+    },
+    {
+      isList: true,
+      kind: referenceForModel(ServiceBindingRequestModel),
+      namespace,
+      prop: 'servicebindingrequests',
+    },
+  );
```

Comment:
You're fetching these resources even if they do not exist. This can cause an error just like we encountered with knative. We need to only include these resources if they are installed and if we have read access. So using `optional: true` here seems like it would solve the issue. Additionally don't include if `serviceBinding === false`